blog/_posts/2018-07-09-$we-init-introductory-post.md

```diff
@@ -0,0 +1,74 @@
+* She loves to make a strict time-table of tasks in hand but rarely follows it.
+* Completely opposite to her partner Rahmeen who has an amazing memeory, she is quite forgetful and would be found digging up stuffs. 
+* She loves working in collaboration with others and is passionate about her work.
+
+
+## What unites us?
+We’re both majoring in Computer Engineering from the same University, Netaji Subhas Institute of Technology, University of Delhi, and are in the same class. We met on Day 1 of college and have been good friends ever since. We started working together when we took an online course on web development. Brainstorming and discussing vividly on assignments and lectures taught us way more than we could have ever imagined. By solving each other’s difficulties and doubts and constantly encouraging the other in their lows, we ensured that neither of us was left behind. 
+
+Working with each other was such an enriching and enjoyable experience that we decided to team up with each other for nearly every project we took- classroom presentations, curriculum centric projects, Hackathons and now RGSoC. It would be no exaggeration to say that our common love for programming  brought us in togetherness and strengthened our friendship. 
+We were sitting next to each other(inseparably as always) attending a tech talk by our most adorable seniors, Prachi Manchanda and Ipshita Chatterjee when we first heard about RGSoC. Immediately we looked at each other and it was clear. We had a new goal, a new dream and what made it special was that we wanted to live it and realise it together.
+
+And then was formulated "$we init" over a plate full of sizzling chilly potatoes at the college canteen. Rahmeen, the creative was suggesting mind blowing names as "OneHotEncoder", "Sopho-no-mores" and Akshita the critic was invariably rejecting them all. And then, together again, they zeroed down to "we're in it", more "terminally" (please say you get the pun) stated as "$we init". Rahmeen couldn't be prouder of "terminalising" the team name thus (It'll break her heart if you didn't appreciate her). 
+
+## Why Probot?
+Having worked with Javascript for almost a year now, we have fallen in deep love with it. NodeJS is one framework that we are extremely passionate about. We have worked with Node on multiple hackathon projects and want to explore it further. 
+
+Even before the project list was announced, we were perusing all the then ‘pending’ projects and that was when we found ‘Probot’ and were blown away by its idea almost immediately. We were completely excited by the concept of building Github Apps to automate github workflow. The fact that even we, who are mere beginners here,  could install these apps on our repos and see the magic happen, was enough to transport us to the Probot slack channel within minutes after the final projects announcement on 1st February. 
```

Comment:
' .. after the final projects announcement ..' sounds bit weird. May be change it to '.. after the announcement of final projects.. '. :)